### PR TITLE
Up font-awesome version to ^6.0.0

### PR DESCRIPTION
**Pull request by iamdarkle**
I have pushed a PR (https://github.com/components/font-awesome/pull/55) for the FA6 update in `components/font-awesome`, the new version is already tagged (https://github.com/components/font-awesome/releases/tag/6.0.0). This PR is just to push the requirement to version 6.0.0.

### CODE REVIEW

**Comment by datitisev** [comment]
So I actually attempted to do this. This is what I found through this process:

- There seems to be no reason to use `components/font-awesome` as `fortawesome/font-awesome` is the official package which provides the same code
- FontAwesome v6 uses `each` in its less which is unsupported by `wikimedia/less.php`, the package that we use to process LESS (#3283)

    ```php
    ParseError: Unexpected input in _sizing.less on line 15, column 5
    13| 
    14| // step-based scale (with alignment)
    15| each(.fa-sizes(), {
    16|   .@{fa-css-prefix}-@{key} {
    17|     .fa-size(@value);
    18|   }
    ```
- There are no up-to-date LESS PHP processors

In short, simply updating to v6 causes errors. And so we have to do something more complex before Flarum 2.0 to be able to update.

**Comment by iamdarkle** [comment]
I hadn't seen that issue, thanks for trying it out, I didn't think it would cause those problems... 😬
I'll keep an eye out for progress on the workaround, which also sounds much better in the long run, so I'll close the PR as it doesn't make much sense.
